src/components/App.svelte

```diff
@@ -42,14 +42,23 @@ $: rememberInfo && localStorage.setItem('additional', additional)
 $: !rememberInfo && localStorage.clear()
 
 async function copy() {
-  const sig = new Blob( [document.querySelector('figure').innerHTML], {type: "text/html"} )
+  const sig = document.querySelector('figure').innerHTML
   
   try {
-    const data = [new ClipboardItem({ "text/html": sig })]
+    const data = [new ClipboardItem({ "text/html": sig }), new ClipboardItem({ "text/plain": sig })]
```

Comment:
Why is `new ClipboardItem({ "text/html": sig })` inserted twice?

Reply:
You'll notice that's one has plain instead of html and it is how it was done in the old method too. I haven't found a case where only using one doesn't work, but Billy said it depends on the email client so I am doing it out of precaution since it doesn't seem to hurt anything.